cmd/dispenser/index.html.tpl

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+  <head>
+    <title>Algorand dispenser</title>
+    <script src='https://www.google.com/recaptcha/api.js'>
+    </script>
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"
+      integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+      crossorigin="anonymous">
+    </script>
+    <script>
+      function sanitize(string) {
+        const entityMap = {
+          '&': '&amp;',
+          '<': '&lt;',
+          '>': '&gt;',
+          '"': '&quot;',
+          "'": '&#39;',
+          '/': '&#x2F;',
+          '`': '&#x60;',
+          '=': '&#x3D;'
+        };
+        return String(string).replace(/[&<>"'`=\/]/g, function (s) {
+          return entityMap[s];
+        });
+      }
+
+      function loadparam() {
+        const queryString = window.location.search;
+        const urlParams = new URLSearchParams(queryString);
+        const account = sanitize(urlParams.get('account'))
+        const pattern = /[A-Z0-9]{58}/
```

Comment:
For the curious: https://github.com/algorand/js-algorand-sdk/blob/master/src/encoding/address.ts#L10